### DISCUSSION TITLE
Cannot install compose (in a restricted environment)

### DISCUSSION THREAD

**Discussion by ericjvandervelden** [podman-desktop/podman-desktop#7744] [Q&A]
I am in a restricted environment. 

I have installed podman desktop on my windows machine and I have a problem with installing Compose. It says: Compose will be downloaded in the next step (Version ${onboardingContext}). Then it says: Failed downloading Compose.

In the logs I see:
Telemetry request error: [object Object]
main ↪️ Error occurred in handler for 'command-registry:executeCommand': Error: An object could not be cloned.

**Comment by ChaosWars** [comment]
Same issue here, on a Citrix managed environment where I do have elevated privileges.

**Reply by benoitf** [reply]
@ChaosWars is that the managed environment has access to github.com ?
